blog_tags.json

```
@@ -519,6 +519,10 @@
                       "database-ssl-primer", "pre-populating-db-connections",
                       "jakartaee9-data-source-transactions-20008-beta", "testing-database-connections-REST-APIs"]
         },
+        {
+            "name": "user story",
```

Our naming convention for blog tags is to use dashes (`-`) and not spaces (and all lower case letters, though not an issue here).  
```suggestion
            "name": "user-story",
```

Also, when I read 'user story' the first thing that comes to mind is Agile process user story.  Given the prevalence of this term in the industry, I'm worried some might get the wrong idea behind the label.  Perhaps another term?
testimonial, reference, user-reference, use-case, case-study, end-user, liberty-user,

---

This is why I wanted your review. :) Thanks. I've changed to `case-study`.